### PR TITLE
Sync similar element of SqlCommand

### PR DESCRIPTION
**Pull request by Wraith2**
A small merge on SqlCommand to sync up similar elements on both sides. This leaves lots of differences in place which will be worked on in followup PR's but this file is too large and too complicated to rework in a single PR.

### CODE REVIEW

**Review by cheenamalhotra — APPROVED** [review]

File: src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlCommand.cs [hunk 1]
```diff
@@ -745,21 +734,15 @@ override public string CommandText
...
-            }
-        }
+        public SqlCommandColumnEncryptionSetting ColumnEncryptionSetting => _columnEncryptionSetting;
 
         /// <include file='../../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlCommand.xml' path='docs/members[@name="SqlCommand"]/CommandTimeout/*'/>
         [
         ResCategoryAttribute(StringsHelper.ResourceNames.DataCategory_Data),
         ResDescriptionAttribute(StringsHelper.ResourceNames.DbCommand_CommandTimeout),
         ]
         override public int CommandTimeout
```
Comment (DavoudEshtehari) [inline src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlCommand.cs:744]: Could you swap `public' and 'override` here too?

File: src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlCommand.cs [hunk 2]
```diff
@@ -1312,13 +1292,7 @@ override protected DbParameter CreateDbParameter()
         override protected void Dispose(bool disposing)
```
Comment (DavoudEshtehari) [inline src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlCommand.cs:1292]: Same here.

**Review by DavoudEshtehari — APPROVED** [review]

File: src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlCommand.cs (hunk 1 above)
Comment (Wraith2) [reply src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlCommand.cs]: Done, and a few others too.

File: src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlCommand.cs (hunk 2 above)
Comment (Wraith2) [reply src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlCommand.cs]: Done.

**Pull request merged**